matRad/doseCalc/+DoseEngines/matRad_ParticleAnalyticalBortfeldEngine.m

```diff
@@ -362,6 +369,7 @@ function calcLateralParticleCutOff(this,cutOffLevel,~)
                 end
             catch
                 msg = 'Your machine file is invalid and does not contain the basic field (meta/data/radiationMode)!';
+                error(msg);
```

Comment:
No we should not throw an error here, as this function will be used to check the availability (and here it just should give the message why it is not available).